integration-tests/integration/cellMove/viewCellSharingRiskAssessmentDetails.spec.js

```diff
@@ -0,0 +1,77 @@
+const offenderFullDetails = require('../../mockApis/responses/offenderFullDetails')
```

Comment:
No fancy Page object? (I like this way better)

Reply:
Actually, sorry I'm blind. I can see it below 👁️

Reply:
That's just the json response, there's a fancy page object underneath I'm afraid :D